TAC/Meetings/2022/2022-02-24/CCC_TAC_Minutes-2022-02-24.md

```diff
@@ -0,0 +1,193 @@
+
+### Outreach update
+
+BW reviewed discussions on the Outreach committee. He noted that Outreach discussed OC3, and provided an update on staffing. He noted the Confidential Computing in Healthcare webinar will be delayed.
+
+DT asked if there are any events with speaking and participation opportunities that would be relevant to projects over the next six months. Nick Vidal (NV) noted that Outreach is working ona spreadsheet with this info and doing specific outreach to projects on this topic. He noted he hopes this will be ready for the next call.
+
+NV requested clarification on the date for the Enarx project review. DT reviewed available times. NV requested March 10, 2022 for the Enarx review.
+
+**Action:** Brian Warner to add Enarx annual review to the agenda for March 10th TAC meeting.
+
+BW noted that DM had joined to discuss Diversity, Civility, and Inclusion.
+
+DT noted that there was a recent discussion with an Andrew Braunberg from Futuriom. DT, Richard Searle, and Ravi Sharma participated. He noted that it was a Q&A, and that Andrew had already seen the whitepaper. He noted that the discussion went well.
+
+DM noted that the D&I discussion was based upon the TAC discussion, and was additionally focused on marketing topics. DM noted that the goal is to provide projects with materials they can use as well.
+
+### Common terminology
+
+Eric Voit (EV) reviewed recent changes to the diagram and noted additional context had been added to the bullet points underneath. He noted that additional context was provided regarding Enarx, where one build could be used to maintain a common north-bound interface, allowing a single project to abstract away the isolation technologies underneath. Finally, he noted descriptive text from Enarx that highlights certain aspects which be abstracted, allowing projects to span multiple isolation technologies.
+
+DT noted that it's helpful to have specific examples from projects, in light of the text about Enarx. He suggested that other projects contribute content to the descriptive text as well, to help place the CCC's role in broader context. EV agreed and noted that contributions from other CCC projects would be welcomed. DT recommended that project mentors involve their projects in contributing text.
+
+Steve Van Lare (SVL) agreed that it would be good to include other projects, as there are different TEE solutions based upon architectures, and that there are differences between encryption and isolation in the tech stacks. 
+
+DT continued discussion on the Anjuna glossary definition of confidential computing. He noted that the Anjuna definition of Confidential Cloud covers protection of data-in-use, data-in-flight, and data-at-rest. He noted that the definition was broader than the CCC scope, which explicitly refers to data-in-use. He noted that he would like the document to cover these other layers as well, in keeping with SVL's original suggestions.
```

Comment:
```suggestion
DT continued discussion on the Anjuna glossary definitions related to confidential computing. He noted that the Anjuna definition of Confidential Cloud covers protection of data-in-use, data-in-flight, and data-at-rest. He noted that the definition was broader than the CCC scope, which explicitly refers to data-in-use. He noted that, speaking as the Microsoft rep, he would like to see the document define one or more terms that cover all three areas, in keeping with SVL's original suggestions.
```
clarified that I argued for one or more terms that span all three legs as prerequisites, not for any terms specific to the other two.

Reply:
Thanks, this is context that I missed. Looks good.